profiler/shakesapp/README.md

```diff
@@ -0,0 +1,32 @@
+# Profiler Shakespeare Application in Go
```

Comment:
Is it intentional that the casing of this heading is not consistent with the next one below?

Reply:
Updated so cases match.